Source/WebCore/platform/ThemeTypes.cpp

```diff
@@ -33,67 +33,69 @@ namespace WebCore {
 TextStream& operator<<(TextStream& ts, SelectionPart selectionPart)
 {
     switch (selectionPart) {
-    case SelectionBackground: ts << "selection-background"; break;
-    case SelectionForeground: ts << "selection-foreground"; break;
+    case SelectionPart::Background: ts << "selection-background"; break;
+    case SelectionPart::Foreground: ts << "selection-foreground"; break;
     }
     return ts;
 }
 
 TextStream& operator<<(TextStream& ts, ThemeFont themeFont)
 {
     switch (themeFont) {
-    case CaptionFont: ts << "caption-font"; break;
-    case IconFont: ts << "icon-font"; break;
-    case MenuFont: ts << "menu-font"; break;
-    case MessageBoxFont: ts << "messagebox-font"; break;
-    case SmallCaptionFont: ts << "small-caption-font"; break;
-    case StatusBarFont: ts << "statusbar-font"; break;
-    case MiniControlFont: ts << "minicontrol-font"; break;
-    case SmallControlFont: ts << "small-control-font"; break;
-    case ControlFont: ts << "control-font"; break;
+    case ThemeFont::CaptionFont: ts << "caption-font"; break;
+    case ThemeFont::IconFont: ts << "icon-font"; break;
+    case ThemeFont::MenuFont: ts << "menu-font"; break;
+    case ThemeFont::MessageBoxFont: ts << "messagebox-font"; break;
+    case ThemeFont::SmallCaptionFont: ts << "small-caption-font"; break;
+    case ThemeFont::StatusBarFont: ts << "statusbar-font"; break;
+    case ThemeFont::MiniControlFont: ts << "minicontrol-font"; break;
+    case ThemeFont::SmallControlFont: ts << "small-control-font"; break;
+    case ThemeFont::ControlFont: ts << "control-font"; break;
     }
     return ts;
 }
 
 TextStream& operator<<(TextStream& ts, ThemeColor themeColor)
 {
+    // Use "using enum" when supported by GCC 11
+    // https://en.cppreference.com/w/cpp/language/enum#Using-enum-declaration
```

Comment:
I think this comment is not necessary unless you transform it in a FIXME with an associated radar and/or bugzilla for tracking it. Otherwise, there is a good chance we will just forget about it.

Reply:
I've removed it. It will apply to every enum class in the future anyway so it's not really pertinent to have it here.